src/main/java/mate/project/exception/CustomGlobalExceptionHandler.java

```diff
@@ -48,6 +48,18 @@ public ResponseEntity<Object> handleEntityNotFoundException(
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex,
+            WebRequest request) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", ZonedDateTime.now());
+        body.put("message", ex.getMessage());
+        body.put("status", HttpStatus.BAD_REQUEST.value());
```

Comment:
The HTTP status code is hardcoded. It would be better to get the status code from the exception, if possible.

Reply:
I am not sure why I am getting comments on code that has been already approved by other mentors? Especially that it doesn't relate to the new functions that have been added in the pull request